ddpui/migrations/0153_metric_definition_and_annotation.py

```
@@ -0,0 +1,138 @@
+"""Migration for MetricDefinition and MetricAnnotation models"""
+
+import django.db.models.deletion
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("ddpui", "0152_reportsnapshot"),
+    ]
```

Ruff (0.15.10)

[warning] Mutable default value for class attribute

(RUF012)